### ISSUE TITLE
Access to the training dataset on Hugging Face

### ISSUE THREAD

**Issue by Acheerfulloser** [Yanhui-Lee/IAD-R1#19]
Hello, author, regarding the classification of defect types in the article, some of them are a bit difficult to distinguish just by their names. May I ask how I can obtain the training dataset? I submitted an application on Hugging Face but only received test set samples, and it seems there is no content in the training set folder.  Email: xiejiahui711@gmail.com

**Comment by Apcin** [comment]
@Acheerfulloser I am also trying to reproduce IAD-R1 but haven't heard back from the authors. If you have the dataset, would you mind sharing it with me at [[6909774@qq.com](mailto:6909774@qq.com)]?
